pkg/config/otelconfig/otelconfig_test.go

```diff
@@ -0,0 +1,85 @@
+package otelconfig
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+)
+
+func TestShouldCreate(t *testing.T) {
+	tests := []struct {
+		opts     v1.Options
+		otelCfg  v1.FreeForm
+		expected bool
+	}{
+		{
+			expected: false,
+			opts:     v1.NewOptions(map[string]interface{}{"config": "/etc/config.yaml"}),
+		},
+		{
```

Comment:
Another like this, but with `config` - and `expected: false`.

Reply:
I am not sure what you mean. I have added some case, I think there are all the combinations now.